spec/support/features/document_capture_step_helper.rb

```diff
@@ -0,0 +1,67 @@
+module DocumentCaptureStepHelper
+  def attach_and_submit_images
+    attach_images
+
+    # If selfie is required, we simulate a submission with the document capture
+    # react component. As a result, the page has already been submitted so
+    # submitting is a noop
+
+    if javascript_enabled? && !selfie_required?
+      click_on 'Submit'
+      # Wait for the background image job to finish and success flash to appear before continuing
+      Capybara.using_wait_time(3) do
+        expect(page).to have_content(t('doc_auth.headings.capture_complete'))
+      end
+    elsif !javascript_enabled?
+      click_idv_continue
+    end
+  end
+
+  def attach_images
+    if javascript_enabled?
+      attach_images_with_js
+    else
+      attach_images_without_js
+    end
+  end
+
+  def attach_images_with_js
+    if selfie_required?
+      simulate_image_upload_api_submission
+    else
+      attach_file 'Front of your ID', 'app/assets/images/logo.png'
+      attach_file 'Back of your ID', 'app/assets/images/logo.png'
+    end
+  end
+
+  def attach_images_without_js
+    attach_file 'doc_auth_front_image', 'app/assets/images/logo.png'
+    attach_file 'doc_auth_back_image', 'app/assets/images/logo.png'
+    attach_file 'doc_auth_selfie_image', 'app/assets/images/logo.png' if selfie_required?
+  end
+
+  def simulate_image_upload_api_submission
+    document_capture_form = page.find('#document-capture-form')
+    session_uuid = document_capture_form['data-document-capture-session-uuid']
+    endpoint = document_capture_form['data-endpoint']
+    connection = Faraday.new(url: URI.join(endpoint, '/')) do |conn|
```

Comment:
It's particular about a trailing slash? 🤔  Also a bit confused why we need to pass a URL both here and below in `connection.post`.

Reply:
So, this is the result of 2 things:

1. The way the Faraday DSL has you setup connections with the multipart middleware
2. Me forgetting to clean this up

Faraday has you initialize it with a base url, and then make requests to a path. `URI.join(endpoint, '/')` is shortcut for turning `https://example.com/abc/123/` into `https://example.com/`.

I feel like there's gotta be a better way to do this. I meant to go back around and figure out a better way to do this once I got it working and let it slip through. Let me take another pass.

Reply:
I just rebuilt this method